src/routers/AppRouter.tsx

```diff
@@ -40,7 +35,7 @@ const getFnrFromParams = (): string => {
 export const dialogmoteRoutePath = "/sykefravaer/dialogmote";
 export const moteoversiktRoutePath = "/sykefravaer/moteoversikt";
 
-const AktivBrukerRouter = (): ReactElement => {
+const AktivBrukerRouter = ({ fnr }: { fnr: string }): ReactElement => {
```

Comment:
Kan man ikke bare gjøre `(fnr: string)` her?

Reply:
Da må vi vel endre til en vanlig funksjon typ `const aktivBrukerRouter = (fnr: string) => {}` i stedet for at det er en React-komponent? Tenker det er bedre å kunne bruke den som en komponent ala `<AktivBrukerRouter fnr={fnr} />`